nibabies/workflows/bold/resampling.py

```diff
@@ -135,48 +199,422 @@ def select_target(subject_id, space):
         name="outputnode",
     )
 
-    # fmt: off
+    # 0, 1 = wm; 2, 3 = pial; 6, 7 = mid
```

Comment:
Since this is such a large addition, I think it would make more sense to make this a subworkflow, and call it within `init_bold_surf_wf` so we don't clutter the connections.